content/chat/patterns/storage.textile

```diff
@@ -0,0 +1,190 @@
+---
+title: Message storage
+meta_description: "Store your messages outside of Ably."
+product: chat
+languages:
+  - javascript
+---
+
+Use of room "history":/chat/rooms/history enables you to retrieve messages stored by Ably from between 72 hours, up to 30 days. In some instances you may wish to store messages for longer than this, or you would prefer to store all messages in your own database.
+
+The motivation for storing messages externally could be for various reasons, such as:
+
+* For privacy reasons.
+* The need to analyze the historic data that users send.
+* So that messages are available for moderation, review or audit purposes.
+* When you require storage longer than 30 days.
+
+Storing message history outside of Ably can be achieved using "Ably Pub/Sub":/products/channels and its integrations.
```

Comment:
I think we should say that it can be achieved using the integrations but in the beta stage, those integrations will have to be configured with the channels which are the pub/sub primitives behind chat features in a room.